docs/reference/announcements.md

```diff
@@ -10,6 +10,25 @@ Scheduled release date: 8th of Oct 2024
 
 Scheduled end of maintenance: 14th of April 2026
 
+### Zeebe repo rename impacts Go client
+
+The Camunda 8 Github repository was renamed from `http://github.com/camunda/zeebe` to `http://github.com/camunda/camunda` on May 23, 2024.
+
+Applications that are already compiled, deployed, and running are **not affected**.
+
+Applications importing the Zeebe Go client under development will fail to compile if referencing the old repository after May 23, 2024. The Zeebe Go Client path should reflect the renamed repo as follows:
```

Comment:
That is not 100% true. 

Maybe we could explain that due to our repo name we renamed also the go module for consistency reasons. 

This means that new versions are provided are under the new name only. In consequence if they want to use a newer version of the client they have to migrate to the new module name.

Does this make sense?

Reply:
Let me see if I'm understanding this correctly. Instead of a date, the driver is the version number? Would this be starting in 8.6.0?

Reply:
Adjusted in the latest commit.

I did not include "Applications that are already compiled, deployed, and running are **not affected**." since the rest of the message assume active development.